### DISCUSSION TITLE
Are Title and Text Component doesnt support auto dark mode?

### DISCUSSION THREAD

**Discussion by masterbater** [mantinedev/mantine#1293] [Q&A] [answered]
During my setup Title Component and Text wont change dynamically with the set color scheme. What are the components that wrap this to have auto dark mode changes that has no padding or any styling. Tried wrapping it in container it doesnt change still black even in dark mode. Though Table Card components do change the color

**Comment by rtivital** [comment, marked as answer]
Title and Text components inherit color from parent element, set `withGlobalStyles` on MantineProvider to set color on body element
